packages/shared/src/logger/logger.ts

```diff
@@ -71,6 +72,7 @@ class Logger {
       return;
     }
 
+    this.grafana.logger.close();
```

Comment:
@miriambudayr I had to add, otherwise, the process wasn't exiting `packages/puppeteer/src/first-run.ts`.

Reply:
Huh, interesting. So basically the CLI process would hang forever?

Reply:
@Andarist also noticed this -> #570

I'm going to merge and resolve conflicts to unblock this branch from being merged :)

Reply:
Noticed that you're calling `close` before `flush` here though. Is that intentional?

If it's important to call both, it seems like flushing _before_ closing would be the expected order. (That's the order @Andarist chose too.)

Reply:
> Noticed that you're calling close before flush here though. Is that intentional?

I specifically chose to call flush before close so we could await the flush. Close also flushes things but it's using a "detached" promise to do that (it's not returned):
https://github.com/JaniAnttonen/winston-loki/blob/2ed0954e97d85e4be1b81f8d853f1864fa37cf6c/src/batcher.js#L304

Reply:
Nice! Good clarification. (Maybe we should add an inline comment with a link to the docs?)